src/main/webapp/list.html

```diff
@@ -1,18 +1,86 @@
 <!DOCTYPE html>
 <meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
 
 <title>ZoomTube</title>
 <link rel="stylesheet" href="style.css" />
+<link
+  rel="stylesheet"
+  href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+  integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+  crossorigin="anonymous"
+/>
 <script src="lectures.js" defer></script>
-<form name="lectureForm" action="/lecture" method="post">
-  <h2>Add a new lecture!</h2>
-  <h3>Enter a name for this lecture:</h3>
-  <input type="text" placeholder="Lecture Name" name="name-input" required />
-  <br />
-  <h3>Enter the YouTube link for this lecture:</h3>
-  <input type="url" placeholder="Lecture URL" name="link-input" required />
-  <br />
-  <br />
-  <button type="submit">Submit</button>
-</form>
-<ul id="lecture-list"></ul>
+
+<body>
+  <nav class="navbar navbar-light">
+    <h2 id="header-text">ZoomTube</h2>
+    <button type="button" class="btn btn-outline-primary">Logout</button>
+  </nav>
+
+  <div id="list-cont">
+    <div id="text-cont">
+      <h2 class="text-center">Active Lectures</h2>
```

Comment:
I find the word "Active" a bit weird. Would "Available Lectures" make more sense? I do not feel strongly against this. If the others like it you can keep it as is.

Reply:
I agree, made the fix